src/translations/nbTranslations.json

```
@@ -717,17 +717,18 @@
   },
   "licenses": {
     "additional_info": {
-      "cc0": "<p>Det bør deles et åpent eller lukket notat, sammen med verket som:</p><ol><li>Gjør rede for konklusjonen om at verket er å oppfatte som et \"fellesgode/public domain\", som definert av Åndsverksloven §11 Opphavsrettens vernetid eller §14 Åndsverk som ikke har opphavsrettslig vern, eller etter en annen lignende bestemmelse</li><li>Dokumenterer at opphaver frasier seg alle opphavsrettigheter og relaterte rettigheter de hadde til verket, som for eksempel omtalt i Åndsverksloven §3 Opphavsrettens innhold og §8 Fellesverk, og at verket er å oppfatte som et \"fellesgode/public domain\"</li></ol>"
+      "cc0": "<p>Se Åndsverksloven <link1>§3, §8, §11 og §14</link1></p>"
```

bedre å ha linken på selve Åndsverksloven, da det er dit den peker?
```suggestion
      "cc0": "<p>Se <link1>Åndsverksloven</link1> §3, §8, §11 og §14</p>"
```